internal/storage/persistence/persistence.go

```diff
@@ -10,13 +10,20 @@ import (
 	"github.com/artefactual-sdps/enduro/internal/storage/types"
 )
 
+type (
+	WorkflowUpdater func(*types.Workflow) (*types.Workflow, error)
+	TaskUpdater     func(*types.Task) (*types.Task, error)
+)
+
 type Storage interface {
 	// AIP.
 	CreateAIP(ctx context.Context, aip *goastorage.AIP) (*goastorage.AIP, error)
 	ListAIPs(ctx context.Context, payload *goastorage.ListAipsPayload) (*goastorage.AIPs, error)
 	ReadAIP(ctx context.Context, aipID uuid.UUID) (*goastorage.AIP, error)
+	AIPDBID(ctx context.Context, aipID uuid.UUID) (int, error)
```

Comment:
Should this method be named `ReadAIPDBID`?

Reply:
Probably, but I'll remove it based on your following comment.